docker/release/base/Dockerfile.in

```diff
@@ -0,0 +1,8 @@
+FROM fedora:34 AS keylime_base
+LABEL version="_version_" description="Keylime Base - Only used as an base image for derived packages"
+MAINTAINER Thore Sommer <mail@thson.de>
```

Comment:
I'm fine if you want to put yourself as the maintainer, but it does mean people might contact you personally if they have problems with keylime (even unrelated to the image itself). It might better to put the keylime project there instead to avoid having that headache.

Reply:
Good point what E-Mail should I put there?

Reply:
how about main@keylime.groups.io

Reply:
Should be fixed in #740.